master/reference/calicoctl/commands/node/run.md

```diff
@@ -71,31 +79,47 @@ Description:
 
 ```
 # Start the Calico node with a specific IPv4 address for BGP.
-$ sudo calicoctl node run --ip=10.10.0.12
+$ sudo dist/calicoctl node run
```

Comment:
why do you need a `dist/` here in user guide? Most users will just use `calicoctl` from their PATH